app/src/test/java/org/simple/clinic/settings/changelanguage/ChangeLanguageEffectHandlerTest.kt

```diff
@@ -99,4 +104,18 @@ class ChangeLanguageEffectHandlerTest {
     verify(uiActions).restartActivity()
     verifyNoMoreInteractions(uiActions)
   }
+
+  @Test
+  fun `when trigger sync effect is received, then sync data`() {
+    // given
+    testCase.dispatch(TriggerSync)
+
+    // when
+    testCase.assertNoOutgoingEvents()
+
+    // then
+    verify(dataSync).fireAndForgetSync(SyncGroup.DAILY)
+    verifyZeroInteractions(uiActions)
```

Comment:
Seems like the statements are placed in the wrong sections. Please check?

Reply:
Yes, fixed it. I have also split the Update and EffectHandler changes into different commits, I have amended by mistake